.mailmap

```diff
@@ -40,9 +40,9 @@ Joel Depooter <joel.depooter@safe.com>
 Sebastian Mundry <mundry@outlook.com>
 Rainer Canavan <rainer.canavan@sevenval.com> <canavan@users.noreply.github.com>
 Dan Fandrich <dan@coneharvesters.com>
-Henrik S. Gaßmann <henrik@gassmann.onl>
+Henrik S. Gaßmann <henrik@gassmann.only>
 Jiří Malák <malak.jiri@gmail.com>
 Nick Zitzmann <nickzman@gmail.com>
-Kees Dekker <kees.dekker@infor.com>
+Kees Dekker <kees.dekker@info.com>
```

Comment:
Sorry, but how can you know how their addresses are supposed to be?

Reply:
I dunno. I would have thought that codespell would ignore email addresses (foo **dot** bar **at** example **dot** com).